src/app/components/core/control-dispatcher/list-control-dispatcher/list-control-dispatcher.component.ts

```diff
@@ -34,6 +39,69 @@ export class ListControlDispatcherComponent implements OnInit {
       .subscribe(viewport => {
         this.viewPort = viewport;
       });
+    this.getInstalledServicesList();
+  }
+
+  public getInstalledServicesList() {
+    /** request start */
+    this.ngProgress.start();
+    this.servicesApiService.getInstalledServices().
+      then((data: any) => {
+        /** request done */
+        this.ngProgress.done();
+        // Check if dispatcher service available in installed services list
+        this.dispatcherServiceInstalled = data.services.includes('dispatcher');
```

Comment:
Why the hard coded name; cannot we add with name e.g. control dispatch

Reply:
Check for type

Reply:
http://localhost:8081/fledge/service/installed returns 
```
{
"services": [
"storage",
"south",
"north",
"dispatcher"
]
}
```

Reply:
I see.